src/NewTwitchApi/Resources/ModerationApi.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+declare(strict_types=1);
+
+namespace NewTwitchApi\Resources;
+
+use GuzzleHttp\Exception\GuzzleException;
+use Psr\Http\Message\ResponseInterface;
+
+class ModerationApi extends AbstractResource
+{
+
+  /**
+  * @throws GuzzleException
```

Comment:
The docblocks in this class should have the first/only asterisk of each line aligned, rather than being aligned with the `/`.